modules/custom/d_p_carousel/config/install/field.storage.paragraph.field_d_cta_single_link.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
```

Comment:
If the field with a universal name and purpose has been created then the storage config should be in a global location, like for the `field_d_cta_link`. Let's move it to the `d_p` modules/feature.

Reply:
I've moved this file to the correct folder.